integration-tests/src/main/java/org/apache/druid/testing/DockerConfigProvider.java

```diff
@@ -202,6 +205,12 @@ public boolean manageKafkaTopic()
       {
         return true;
       }
+
+      @Override
+      public String getExtraDatasourceNameSuffix()
+      {
+        return extraDatasourceNameSuffix;
```

Comment:
Do you have a use case to make the suffix configurable ? 
If not we can just have the suffix in the test datasource name itself, i believe it would simplify the changes. and each test can choose its own datasource name format.

Reply:
My thinking was that the non-ASCII characters could make the tests fail for people who do not have the proper locales set up on their system. I had to tweak locale settings on the ubuntu containers used in the integration tests, and people may not have any need for such characters in their own development/use cases, so I felt it would be nice to have a way to disable the use of such characters in the tests.

Another reason for making it a "global" thing like this is because I wanted to easily test support across all the use cases being tested in the IT suite (e.g., if I wanted to test support for some other characters across the board, I can just edit this property instead of changing each test individually)